e2e-playwright/plugin-e2e/plugin-e2e-api-tests/as-admin-user/panelDataAssertion.spec.ts

```diff
@@ -38,7 +38,7 @@ test.describe(
           formatExpectError('Could not locate header elements in table panel')
         ).toContainText(['col1', 'col2']);
         await expect(
-          panelEditPage.panel.locator.getByRole('gridcell'),
+          panelEditPage.panel.data,
```

Comment:
Where will this look for the values? I'm not familiar with what the `data` locator is? At first glance seems like a broader locator than the previous `gridcell` that seemed more specific.